CODEOWNERS

```diff
@@ -54,6 +54,12 @@ pnpm-lock.yaml @nrwl/nx-core-reviewers
 /packages/react-native/** @nrwl/nx-react-reviewers
 /e2e/react-native/** @nrwl/nx-react-reviewers
 
+# Vue
+/docs/generated/packages/vue/** @nrwl/nx-js-reviewers @nrwl/nx-docs-reviewers
+/docs/shared/packages/vue/** @nrwl/nx-js-reviewers @nrwl/nx-docs-reviewers
+/packages/vue/** @nrwl/nx-js-reviewers
+/e2e/vue/** @nrwl/nx-js-reviewers
```

Comment:
Can we go ahead and make this a separate team? I can create it if you update the code